### ISSUE TITLE
Order popular discover cards by count

### ISSUE THREAD

**Issue by carlbennetts** [status-im/status-react#678]
Discover tags should be ordered by count.

Example: Right now a less popular tag may appear to a user before more popular tags.

<img src="https://s3.amazonaws.com/instabug-pro/states/files/44464808/screenshots/ebe13d54cfc1dcb5b218b8c214bcac53_original.jpg?1483604909" width="320">

**Comment by jumarko** [comment]
Hi, I'm about to start to work on this issue. Please, set me as an assignee and set the "in-progress" label.

FYI: I've just discussed the proper "issue assignment process" with @carlbennetts. We've agreed that he will check with @jarradh and let me know what's the proper process to start work on issue.

**Comment by jarradh** [comment]
set as in-progress

**Comment by jarradh** [comment]
any updates on this @jumarko ?

**Comment by jumarko** [comment]
@jarradh well, I've managed to build and run the status app and play with it a little bit.
I haven't gotten into the details (code) yet, but I expect to look at it tomorrow.

**Comment by jumarko** [comment]
@jarradh I'm not sure how to simulate this bug.
I tried to do check with a single user changing status to multiple values but I always end up with proper ordering (status with the biggest count ordered first).
It might be that my test scenario is different from reality or that you had some special setup.

I looked into the code and at first sight it looks good:

https://github.com/status-im/status-react/blob/develop/src/status_im/discover/handlers.cljs#L121
```
(defn reload-tags!
  [db _]
  (assoc db :tags (discoveries/get-all-tags)
...
```

https://github.com/status-im/status-react/blob/develop/src/status_im/data_store/realm/discover.cljs#L78
```
(defn get-all-tags []
  (-> (realm/get-all @realm/account-realm :tag)
      (realm/sorted :count :desc)
      realm/realm-collection->list))
```

Therefore, tags should be sorted by count descending.

Could you give me some hints how to reproduce this issue?

**Comment by jumarko** [comment]
@jarradh Step-by-step reproducer would definitely help. Until then I don't think I can do much with this issue.

**Comment by jumarko** [comment]
@alwx , would you mind checking if it still happens and if so, giving me some guidance how to repreduce it? Thank you.

**Comment by alwx** [comment]
@jumarko everything seems to be fine — I was unable to reproduce this issue. But while trying, I was able to find another one — #778. Probably there is some relation between this tasks, and probably the reason of incorrect ordering somehow related to the problem with updates. Please, check the new issue and tell me what do you think about it.

**Comment by jumarko** [comment]
@alwx I'd suggest to close this issue if it cannot be reproduced.
Please, take a look at https://github.com/status-im/status-react/issues/778 too. I've found the behavior to be appropriate so I wonder if we should close that one too. I also asked @jarradh to take a look at it.

**Comment by goranjovic** [comment]
Solved in #1874
